src/main/java/Deadline.java

```diff
@@ -0,0 +1,24 @@
+public class Deadline extends Task{
+    protected String by;
+    protected static String TYPE = "deadline";
```

Comment:
Is there a reason why the type of tasks is represented even though there is no reference to it?